gridData/OpenDX.py

```diff
@@ -460,6 +483,13 @@ def __init__(self,classid='0',components=None,comments=None):
         self.components = components
         self.comments= comments
 
+    def _openfile(self, filename):
+        """Returns a regular or gz file descriptor"""
+        if filename.endswith('.gz'):
```

Comment:
If you allow bz2 then that also needs to be available when writing.

Reply:
makes sense, I think that I'll remove it for now...